templates/resources/rego_policy_instance.md.tmpl

```diff
@@ -14,25 +14,46 @@ All templates use parameters defined as JSON, below is a list of all the corresp
+- `dbAccounts` (Object) Defines database accounts to include or exclude from the policy. Excluded accounts are not subject to the policy, while included accounts must adhere to it. See [dbAccounts](#objects--dbAccounts).
+- `alertSeverity` (String) Policy action to alert, using the respective severity. Allowed values are: `low`, `medium`, `high`.
 
-### Service Account Abuse (service-account-abuse)
+### Service Account Abuse (service-account-abuse) - Ensure service accounts can only be used by intended applications.
 
 -   `block` (Boolean) Policy action to enforce.
 -   `serviceAccounts` (Array) Service accounts for which end user attribution is always required.
 -   `alertSeverity` (String) Policy action to alert, using the respective severity. Allowed values are: `low`, `medium`, `high`.
 
+### Stored Procedure Governance (stored-procedure-governance) - Restrict execution of stored procedures.
+- `governedProcedures` (Array) List of stored procedures to be governed.
+- `enforce` (Boolean) Whether to enforce the policy, if false, only alerts will be raised on policy violations.
+- `identities` (Object) Defines users, groups, or emails that are included or excluded from the policy. If included identities are defined, only those users are exempt from policy enforcement. Excluded identities are always subject to the policy. See [identities](#objects--identities).
+- `dbAccounts` (Object) Defines database accounts to include or exclude from the policy. Excluded accounts are not subject to the policy, while included accounts must adhere to it. See [dbAccounts](#objects--dbAccounts).
+- `alertSeverity` (String) Policy action to alert, using the respective severity. Allowed values are: `low`, `medium`, `high`.
+
+### Ungoverned Statements (ungoverned-statements) - Control execution of statements not governed by other policies.
+- `block` (Boolean) Indicates whether unauthorized operations should be blocked. If true, operations violating the policy are prevented.
+- `identities` (Object) Defines users, groups, or emails that are included or excluded from the policy. If included identities are defined, only those users are exempt from policy enforcement. Excluded identities are always subject to the policy. See [identities](#objects--identities).
+- `dbAccounts` (Object) Defines database accounts to include or exclude from the policy. Excluded accounts are not subject to the policy, while included accounts must adhere to it. See [dbAccounts](#objects--dbAccounts).
+- `alertSeverity` (String) Policy action to alert, using the respective severity. Allowed values are: `low`, `medium`, `high`.
+
 ### Deprecated policy templates
-The remaining list of policy templates are deprecated an can not be used for creating new policies.
+The remaining list of policy templates are deprecated and can not be used for creating new policies.
```

Comment:
for control planes version 4.18 and beyond/above?

Reply:
done